webextensions/api/browserAction.json

```
@@ -364,7 +364,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/openPopup",
             "support": {
               "chrome": {
-                "version_added": false
+                "version_added": 67
```

```suggestion
                "version_added": "67"
```

Version numbers must be in strings as to allow semantic versioning support.